### ISSUE TITLE
camelcase lint error on module import when deconstructing

### ISSUE THREAD

**Issue by lordgiotto** [eslint/eslint#6755]
**ESLint version:** 

3.1.1
___
**Rule:** 

`"camelcase": [2, { "properties": "never" }]` 
___
**Problem**

Lint fails on module import when deconstructing  properties and aliasing.

``` javascript
// this cause lint error
import { no_camel_case_prop as CamelCaseProp } from 'external-module';
```

But with 186e8f0a555701f02ba0207e8e71b3d941de665a that rule should ignore camelcase during deconstructing.

_(I know that import deconstructing is not a real object deconstructing, but probably it should follow the same rule)_

For now I have to workaround this issue this way:

``` javascript
// this not cause lint error
import * as externalModule from 'external-module';
externalModule.no_camel_case_prop
```
___
Thank you :)

**Comment by vitorbal** [comment]
Hi @lordgiotto, thanks for reaching out!

Just verified that this is true. Even without the `properties` option set to `never`, I would expect the example code to not cause a lint error, for the same reasons outlined in https://github.com/eslint/eslint/issues/3185 for object destructuring.

So, to recap:

**Valid**:
```javascript
/* eslint camelcase: 2*/
import { no_camel_case_prop as CamelCaseProp } from 'external-module';
```

**Invalid**:
```javascript
/* eslint camelcase: 2*/
import { no_camel_case_prop } from 'external-module';
```

**Invalid**:
```javascript
/* eslint camelcase: 2*/
import { no_camel_case_prop as no_camel_case_prop } from 'external-module';
```

@lordgiotto Would this be something you are willing to send a PR for?

**Comment by michaelficarra** [comment]
@lordgiotto The workaround you listed is wrong. It would be

```js
import * as externalModule from 'external-module';
externalModule.no_camel_case_prop;
```

**Comment by lordgiotto** [comment]
@michaelficarra Oh, sorry, you're right! I fix it now.

@vitorbal It would be a pleasure, but to be honest I never read ESLint source code and I should at least study how it's designed before be able to send a PR. So I could do it, but i would need some time :P

**Comment by lordgiotto** [comment]
@vitorbal PR done :)
